### ISSUE TITLE
Libreoffice fails to start

### ISSUE THREAD

**Issue by haunma** [void-linux/void-packages#14581]
### System

* xuname:  
Void 5.2.15_1 x86_64-musl GenuineIntel uptodate rF

* package:
libreoffice-calc-6.2.5.2_2
libreoffice-common-6.2.5.2_2

### Expected behavior
Application starts normally

### Actual behavior
Running "libreoffice" from a command window or from the lxqt menu system results in a brief flash of the splash screen, then nothing.  In the command window, the following messages are displayed:
```
terminate called after throwing an instance of 'com::sun::star::uno::RuntimeException'


Fatal exception: Signal 6
```

### Steps to reproduce the behavior
1) Install libreoffice.
2) Attempt to run it.

I would appreciate pointers on how to debug this further.  The application does not appear to perform any logging, and doesn't have any command-line switches to increase verbosity, that I can see.  This is the first time that I have attempted to run openoffice/libreoffice (on any system).

**Comment by haunma** [comment]
I did not install the virtual package ("libreoffice"), only libreoffice-common and libreoffice-calc, reasoning that those were the ones I needed to get a spreadsheet.  (I'm not interested in the other components.)  Did I miss something else important?

**Comment by Johnnynator** [comment]
It ideally should run with put having the full suite installed, did you have at least one language pack installed?

**Comment by haunma** [comment]
I did some more experimenting, and it seems libreoffice-writer is critical.  I couldn't get libreoffice to start up without it.  (I had already added libreoffice-fonts besides my language pack, libreoffice-common, and libreoffice-calc.  Not sure if the fonts are required or not.)  Anyway, I don't mind having writer for the times I want to open up Word docs.  Probably better than my old standby, antiword.
